report/report.tex

```diff
@@ -119,7 +119,18 @@ \section{Architektur}
 \section{Externe Schnittstellen}
 
 \section{Code}
-
+\subsection{DoD}
+\begin{itemize}
+\item Alle Akzeptanzkriterien werden erfüllt
+\item Der Code ist fertiggestellt und im Versionierungssystem eingespielt
+\item Dokumentation Update durchgeführt
+\item Release Dokumentation angepasst
+\item Es wurde ein Code Review durchgeführt oder der Code wurde im Pair Programming erarbeitet
+\item Coding Guidelines und Standards wurden eingehalten
+\item Es wurden Unit Tests erfolgreich durchgeführt und Testabdeckung erfüllt Kriterien (85\% oder mehr) 
+\item Es sind keine kritischen Bugs offen
+\item "Functional Tests" erfolgreich durchlaufen
```

Comment:
Was sind "Functional Tests"?

Reply:
manual testing im gegensatz zu unit tests. feeding input examining output... durchgeführt von po oder tester.

Reply:
Müssen wir dann für jede Story ein Set an "Functional Tests" definieren?

Reply:
Ich glaube für den Anfang können wir es bei Unit Tests belassen. Können wir nicht in der Retro schauen, ob das erweitert werden soll?

Reply:
Denke auch, dass Unit- und Integrationtests für den Anfang ausreichen